go.mod

```diff
@@ -322,3 +322,5 @@ replace (
 	sourcegraph.com/sourcegraph/appdash => github.com/sourcegraph/appdash v0.0.0-20190731080439-ebfcffb1b5c0
 	sourcegraph.com/sourcegraph/appdash-data => github.com/sourcegraph/appdash-data v0.0.0-20151005221446-73f23eafcf67
 )
+
+replace github.com/tikv/client-go/v2 => github.com/crazycs520/client-go/v2 v2.0.0-alpha.0.20240425073930-b36d4a84d7cb
```

Comment:
Why not use "github.com/tikv/client-go" directly?

Reply:
Since https://github.com/tikv/client-go/pull/1300 isn't merge.